decidim-elections/app/views/decidim/elections/votes/_onboarding_modal.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="reveal" data-reveal id="onboarding-modal">
+  <div class="reveal__header">
+    <h3 class="reveal__title"><%= t("decidim.elections.votes.onboarding_modal.title") %></h3>
+    <button class="close-button" data-close aria-label="Close modal" type="button">
```

Comment:
👀 This button is missing the translation for the `aria-label`:

```suggestion
    <button class="close-button" data-close aria-label="<%= t("decidim.elections.votes.onboarding_modal.close_modal") %>" type="button">
```

Reply:
Oh wow. Great! I'll update it!